modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSearch.testcase

```diff
@@ -1231,6 +1231,8 @@ definition {
 			locator1 = "CommerceAccelerators#SORT_BY_DROPDOWN",
 			value1 = "Sort By: Relevance");
 
+		var productId = ${productId} - 1;
```

Comment:
We've spoken offline, please use data-qa-id

Reply:
Did it, thanks!